src/main/resources/FormItemProfile.json

```diff
@@ -5,6 +5,7 @@
       {
         "fieldName": "Order",
         "metadataPropertyName": "order",
+        "description": "Numerical order for questions in this section",
```

Comment:
We have by default a Mauro index/order property on Data Elements which represents 'the order of this Element in the containing Data Class'. However I added this to the profile to handle the case where a section contains interleaved questions and sub-sections, so I think this should be "Numerical order for items in this section" (or using the language from the ISO standard, possibly "Numerical order for form design elements in this section").

Reply:
Agreed.  Have changed.